### PR TITLE
DSP-1168 Update contribution documentation

### PR DESCRIPTION
**Pull request by kilchenmann**
Updates the contribution documentation because of the integration from DSP-1168

### CODE REVIEW

File: docs/developers/dsp/contribution/index.md [hunk 1]
```diff
@@ -122,17 +125,9 @@ With each push to GitHub, the tests of the repository are executed. Successfull
...
-
-### Release
-
-To make a real release, we have to publish the [release draft](#release-notes), mentioned above, manually. Be sure it's up to date; you have to wait until the release-drafter were run in Github actions and wait until the tests pass on repository's main branch. Update the tag and the release title with the release version number, including the prefix `v`: e.g. v1.0.0 or v1.0.0-rc.0
+### Prepare release (new in DSP-JS, DSP-UI and DSP-App)
 
-Do not forget to check the box "This is a pre-release" in case of a release candidate (-rc.).
+We use [release-please-action](https://github.com/marketplace/actions/release-please-action) (in DSP-JS, DSP-UI and DSP-App) to prepare next release. This action script automates CHANGELOG generation, the creation of GitHub releases, and version bumps. For doing so, it creates a release PR which updates itself with each push into main branch following the commit messages. It's important to use the defined rules from [above](#git-commit-guidelines). When merging this release PR a new release will be created automatically.
```
Comment (mdelez) [inline docs/developers/dsp/contribution/index.md:130]: We use [release-please-action](https://github.com/marketplace/actions/release-please-action) (in DSP-JS, DSP-UI and DSP-App) to prepare next **the** release. This action script automates **the** CHANGELOG generation, the creation of GitHub releases, and version bumps. **In** doing so, it creates a release PR which updates itself with each push into main branch following the commit messages. It's important to use the defined rules from [above](#git-commit-guidelines). When merging this release PR a new release will be created automatically.

File: docs/developers/dsp/contribution/index.md [hunk 2]
```diff
@@ -143,3 +138,20 @@ With each published (pre-)release, the action workflow builds the npm package or
...
+
+After each push into the main branch &mdash; after each merge from a pull request &mdash; the release notes for the next release are updated. This release called "Next release" is a draft and can be used to publish the [real release](#release) later.
+
+The GitHub action we use for this step is [release-drafter](https://github.com/marketplace/actions/release-drafter).
+
+### Release (deprecated)
+
+> :warning: The following release process is deprecated; only used in DSP-API
+
+To make a real release, we have to publish the [release draft](#release-notes), mentioned above, manually. Be sure it's up to date; you have to wait until the release-drafter were run in Github actions and wait until the tests pass on repository's main branch. Update the tag and the release title with the release version number, including the prefix `v`: e.g. v1.0.0 or v1.0.0-rc.0
```
Comment (mdelez) [inline docs/developers/dsp/contribution/index.md:154]: **has** run

**Review by mdelez — CHANGES_REQUESTED** [review]
a few little corrections :)

File: docs/developers/dsp/contribution/index.md (hunk 1 above)
Comment (kilchenmann) [reply docs/developers/dsp/contribution/index.md]: Updated in 7e8119c

File: docs/developers/dsp/contribution/index.md (hunk 2 above)
Comment (kilchenmann) [reply docs/developers/dsp/contribution/index.md]: Fixed in 7e8119c

**Review by mdelez — APPROVED** [review]

File: docs/developers/dsp/contribution/index.md [hunk 3]
```diff
@@ -122,17 +125,9 @@ With each push to GitHub, the tests of the repository are executed. Successfull
 [![DSP-APP CI](https://img.shields.io/github/workflow/status/dasch-swiss/dsp-app/CI?label=DSP-APP%20CI)](https://github.com/dasch-swiss/dsp-app/actions)
 [![DSP-Docs CI](https://img.shields.io/github/workflow/status/dasch-swiss/dsp-docs/Publish?label=DSP-Docs%20CI)](https://github.com/dasch-swiss/dsp-docs/actions)
 
-### Release notes
+### Prepare release (new in DSP-JS, DSP-UI and DSP-App)
```
Comment (flavens) [inline docs/developers/dsp/contribution/index.md:128]: DSP-App should be written "DSP-APP", according to YouTrack, GitHub readmes, the version matrix, etc.
same line 130

**Review by flavens — COMMENTED** [review]
See my comment.
I would also like to have more details about the "Make a draft" section, how to create a PR as a draft and [how to convert an open PR to a draft](https://github.blog/changelog/2020-04-08-convert-pull-request-to-draft/) (possible but not easy to find).
And I think there are some English typo left in the existing text content.

**Comment by kilchenmann** [comment]
> I would also like to have more details about the "Make a draft" section, how to create a PR as a draft and [how to convert an open PR to a draft](https://github.blog/changelog/2020-04-08-convert-pull-request-to-draft/) (possible but not easy to find).

I totally agree, but the mentioned section is not part of this PR. @flavens feel free to update the documentation about the draft section. A link to the github documentation should be enough.

**Comment by kilchenmann** [comment]
> > I would also like to have more details about the "Make a draft" section, how to create a PR as a draft and [how to convert an open PR to a draft](https://github.blog/changelog/2020-04-08-convert-pull-request-to-draft/) (possible but not easy to find).
> 
> I totally agree, but the mentioned section is not part of this PR. @flavens feel free to update the documentation about the draft section. A link to the github documentation should be enough.

I added now two links to the "PR draft" section (e7ba2dc)

File: docs/developers/dsp/contribution/index.md (hunk 3 above)
Comment (kilchenmann) [reply docs/developers/dsp/contribution/index.md]: Resolved in e7ba2dc

**Review by flavens — APPROVED** [review]

**Pull request merged**